waspc/cli/src/Wasp/Cli/Command/CreateNewProject/ArgumentsParser.hs

```diff
@@ -33,7 +33,7 @@ parseNewProjectArgs newArgs = parserResultToEither $ execParserPure defaultPrefs
       Opt.strOption $
         Opt.long "template"
           <> Opt.short 't'
-          <> Opt.metavar "TEMPLATE_NAME"
+          <> Opt.metavar "TEMPLATE"
```

Comment:
Do we maybe want to go with `TEMPLATE_ID` to keep it consistent with the record above?